gradle.properties

```diff
@@ -1,3 +1,7 @@
 org.gradle.parallel=true
 org.gradle.configureondemand=true
 org.gradle.jvmargs=-Xmx256m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+
```

Comment:
If you're added `org.gradle.caching=true` can you please also add `org.gradle.vfs.watch=true` configuration?

Reply:
done